addons/l10n_es_edi_verifactu/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+l10n_es_edi_verifactu.access_l10n_es_edi_verifactu_document_readonly,access_l10n_es_edi_verifactu_document_readonly,l10n_es_edi_verifactu.model_l10n_es_edi_verifactu_document,base.group_user,1,0,0,0
+l10n_es_edi_verifactu.access_l10n_es_edi_verifactu_document_group_invoice,access_l10n_es_edi_verifactu_document_group_invoice,l10n_es_edi_verifactu.model_l10n_es_edi_verifactu_document,account.group_account_invoice,1,1,1,1
```

Comment:
not sure this is the best thing...  (if you check the certificate module in master, it is only to group_system)